src/lib/app/constants/streams.ts

```diff
@@ -4,3 +4,4 @@ export const DEFAULT_MESSAGES_PAGE_SIZE = 5
 export const DEFAULT_AUTHOR_FILTER_STATE = true
 export const DEFAULT_SUBSCRIBED_FILTER_STATE = false
 export const DEFAULT_REQUESTED_SUBSCRIPTION_STATE = false
+export const MESSAGE_EXPLORER = import.meta.env.VITE_MESSAGE_EXPLORER as string
```

Comment:
we could set a default value to 'https://explorer.iota.org/mainnet'. wdyt?

Reply:
good idea. would be better. I guess the url will not change for a long time.